storage/badger/dkg_state.go

```diff
@@ -155,14 +156,14 @@ func (keys *SafeBeaconPrivateKeys) RetrieveMyBeaconPrivateKey(epochCounter uint6
 		}
 
 		// for any end state besides success and recovery, the key is not safe
-		if endState == flow.DKGEndStateSuccess || endState == flow.DKGEndStateRecovered {
-			// retrieve the key - any storage error (including not found) is an exception
+		if endState == flow.DKGEndStateSuccess || endState == flow.RandomBeaconKeyRecovered {
+			// retrieve the key - any storage error (including `storage.ErrNotFound`) is an exception
 			var encodableKey *encodable.RandomBeaconPrivKey
 			encodableKey, err = keys.state.retrieveKeyTx(epochCounter)(txn)
 			if err != nil {
 				key = nil
 				safe = false
-				return fmt.Errorf("[unexpected] could not retrieve beacon key for epoch %d with successful DKG: %v", epochCounter, err)
+				return irrecoverable.NewExceptionf("[unexpected] could not retrieve beacon key for epoch %d with successful DKG: %v", epochCounter, err)
```

Comment:
```suggestion
				return irrecoverable.NewExceptionf("could not retrieve beacon key for epoch %d with successful DKG: %v", epochCounter, err)
```
using `NewExceptionf` will already put a scary prefix on the message - no need for 2.